examples/step-85/doc/intro.dox

```diff
@@ -0,0 +1,246 @@
+                  - (\partial_n u_h, v_h)_\Gamma
+                  - (u_h, \partial_n v_h)_\Gamma
+                  + \left (\frac{\gamma_D}{h} u_h, v_h \right )_\Gamma,
+  \\
+  L_h(v_h)      &=  (f,v)_\Omega
+                  + \left (u_D, \frac{\gamma_D}{h} v_h -\partial_n v_h \right )_\Gamma.
+@f}
+In this formulation, there is one big difference,
+compared to a standard boundary-fitted finite element method.
+On each cell,
+we need to integrate over the part of the domain and the part of the boundary that falls within the cell.
+Thus, on each cell intersected by $\Gamma$,
+we need special quadrature rules that only integrate over these parts of the cell,
+that is, over $T \cap \Omega$ and $T \cap \Gamma$.
+@image html immersed_quadratures.svg
+Since $\Omega \cap T$ is the part of the cell that lies inside the domain,
+we shall refer to the following regions
+@f{align*}
+  \{x \in T : \psi(x) < 0 \}, \\
+  \{x \in T : \psi(x) > 0 \}, \\
+  \{x \in T : \psi(x) = 0 \},
+@f}
+as the "inside", "outside" and the "surface region" of the cell $T$.
+
+The above method is sometimes referred to as the "naive weak formulation"
+because it suffers from the so-called "small cut problem".
+Depending on how $\Omega$ is located relative to $\mathcal{T}_h$,
+a cut between a cell, $T \in \mathcal{T}_h$, and $\Omega$ can be made arbitrarily small:
+$|\Omega \cap T | \rightarrow 0$.
+For Neumann boundary conditions,
+the consequence is that the stiffness matrix can become arbitrarily ill-conditioned
+as the cut-size approaches zero.
+For a Dirichlet condition, the situation is even worse.
+For any finite choice of Nitsche constant, $\gamma_D$,
+the bilinear form $a_h(\cdot,\cdot)$ loose coercivity as the size of a approaches zero.
+This makes the above weak formulation essentially useless
+because as we refine we typically can not control how the cells intersect $\Gamma$.
+One way to avoid this problem is to add a so-called ghost penalty term, $g_h$,
+to the weak formulation (see e.g. @cite burman_hansbo_2012 and @cite cutfem_2015).
+This leads to the stabilized cut finite element method,
+which reads: Find $u_h \in V_h$ such that
+@f{equation*}
+  A_h(u_h, v_h) = L_h(v_h), \quad \forall v_h \in V_\Omega^h,
+@f}
+where
+@f{equation*}
+  A_h(u_h,v_h) = a_h(u_h,v_h) + g_h(u_h, v_h).
+@f}
+The point of this ghost penalty is that it makes the numerical method essentially independent
+of how $\Omega$ relates to the background mesh.
+In particular, $A_h$ can be shown to be continuous and coercive,
+with constants that do not depend on how $\Omega$ intersects $\mathcal{T}^h$.
+To define the ghost penalty, let $\mathcal{T}_\Gamma^h$ be the set of intersected cells:
+@f{equation*}
+  \mathcal{T}_{\Gamma}^h = \{ T \in \mathcal{T}_{\Omega}^{h} : T \cap \Gamma \neq \emptyset \},
+@f}
+and let $\mathcal{F}_h$ denote the interior faces of the intersected cells in the active mesh:
+@f{equation*}
+  \mathcal{F}_h = \{ F = \overline{T}_+ \cap \overline{T}_- : \,
```

Comment:
What is the meaning of the overline?

Reply:
I don't know if deal.II has a convention for this, but some books define T as an open set (others as a closed set). If T_1 & T_2 are open sets, F is empty. If we add the closure it is correct regardless of how it is defined.